.github/dependabot.yml

```diff
@@ -3,5 +3,8 @@ updates:
 - package-ecosystem: npm
   directory: "/"
   schedule:
-    interval: daily
-  open-pull-requests-limit: 20
+    interval: monthly
+  ignore:
+    - dependency-name: "*"
+      update-types: ["version-update:semver-patch"]
```

Comment:
See: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#specifying-dependencies-and-versions-to-ignore